bin/main/org/fundacionjala/at15/katas/README.md

```diff
@@ -0,0 +1,43 @@
+# Katas folder
```

Comment:
Hi @dulphyj , please remove `bin/` folder (`git rm -r bin/`) and add it to `.gitignore`.